mac/Focalboard/ViewController.swift

```diff
@@ -107,8 +107,14 @@ class ViewController:
 
 	private func updateSessionTokenAndUserSettings() {
 		let appDelegate = NSApplication.shared.delegate as! AppDelegate
+		let url = URL(string: "http://localhost:\(appDelegate.serverPort)/")!
+		let sessionScript = """
+		if (window.location.href.toLowerCase().startsWith('\(url)'.toLowerCase())) {
```

Comment:
The .toLowerCase() should not strictly be needed, but is there for future-proofness.